Enterprise-CMCS/macpro-mdct-mcr
# [MLR-2215] Releasing report E2E tests

## Summary
This PR adds Cypress e2e testing for releasing reports:

### Description
Test scenarios:
* I create/fill/submit and release a report with the correct data pre-filled
* I cannot release a submitted and archived report
* I cannot release a non-locked (non-submitted) report.

### Related ticket
MDCT-2215

### How to test
Pull down the repo and run the Cypress tests—or just verify that this PR doesn't fail PR hooks.

### Important updates
Exported the `traverseRoutes` function from the form e2e test so I could use it.

### Author checklist
